autoload/arduino.vim

```diff
@@ -78,6 +75,9 @@ function! arduino#InitializeConfig() abort
   endif
   if !exists('g:arduino_use_cli')
     let g:arduino_use_cli = s:has_cli
+    if g:arduino_use_cli
+      let g:arduino_serial_cmd = 'arduino-cli monitor -p {port} --config baudrate={baud}'
+    endif
```

Comment:
I would move this into the `if !exists('g:arduino_serial_cmd')` above (line 49). Move that check below this one, and then set it to be the default serial command if `g:arduino_use_cli`